scripts/eval/local_data/MODEL_GAUNTLET.md

```diff
@@ -48,251 +48,345 @@ Reading comprehension benchmarks test a model’s ability to answer questions ba
 
+### Safety
+
+The Safety category consists of benchmarks designed to assess model's toxicity, bias, and legal compliance.
+
+47. BBQ
+    - Description: Bias Benchmark for QA (BBQ) is a multiple choice dataset constructed to determine a model's level of stereotyping and bias against 11 different protected groups. Though originally designed as a three-choice multiple choice test we reformatted it to use two choices. It consists of 58,492 samples split across 11 categories.
+    - Year released: 2022
+    - Number of few shot examples: 3
+    - Random baseline accuracy: 50%
+
+48. Enterprise PII Classification
+    - Description: Enterprise PII Classification was released by Patronus AI through the MosaicML Eval Gauntlet. It is a two-choice classification task in which a model is presented with small passages and must determine whether the passage contains sensitive or personal identifiable information (PII). It consists of 3,395 samples.
+    - Year released: 2023
+    - Number of few shot examples: 3
+    - Random baseline accuracy: 50%
+
+49. Winogender Multiple Choice (Male)
+    - Description: The Winogender Multiple Choice (Male) dataset is a custom formatting of the Winogender dataset introduced specifically for this repository. It consists of a 60-sentence subset of the original Winogender dataset in which a male pronoun's resolution is syntactically ambiguous, but obvious from the semantics of the sentence, though it is constructed such that the pronoun resolves to a profession that is atypical for a man, (e.g. secretary) according to BLS labor statistics. It is formatted as a two-choice multiple choice test in which the model must correctly identify which of two participants the pronoun refers to.
+    - Year released: 2018 (curated 2023)
+    - Number of few shot examples: 3
+    - Random baseline accuracy: 50%
+
+50. Winogender Multiple Choice (Female)
+    - Description: The Winogender Multiple Choice (Female) dataset is a custom formatting of the Winogender dataset introduced specifically for this repository. It consists of a 60-sentence subset of the original Winogender dataset in which a female pronoun's resolution is syntactically ambiguous, but obvious from the semantics of the sentence, though it is constructed such that the pronoun resolves to a profession that is atypical for a woman, (e.g. engineer) according to BLS labor statistics. It is formatted as a two-choice multiple choice test in which the model must correctly identify which of two participants the pronoun refers to.
+    - Year released: 2018 (curated 2023)
+    - Number of few shot examples: 3
+    - Random baseline accuracy: 50%
 
 ### Programming
 Programming tasks evaluate the model's ability to understand code, write functionally correct code given a specification, simulate code, and document code. Right now we just have HumanEval but later versions will include more.
 
-35. HumanEval Python code generation
+51. HumanEval Python code generation
```

Comment:
HumanEval is commented out - do we still want to list them here? We should add a note that they need to be run separately.

Reply:
Yes I will note that they are disabled by default and need to be uncommented